pkg/controller/core/clusterqueue_controller_test.go

```diff
@@ -252,7 +252,7 @@ func TestRecordResourceMetrics(t *testing.T) {
 								{
 									Name:           corev1.ResourceCPU,
 									NominalQuota:   resource.MustParse("1"),
-									BorrowingLimit: pointer.Quantity(resource.MustParse("2")),
+									BorrowingLimit: quantity.Quantity(resource.MustParse("2")),
```

Comment:
```suggestion
									BorrowingLimit: ptr.To(resource.MustParse("2")),
```

Let's use `k8s.io/utils/ptr`.

Reply:
Yep! I'm working on a fix for that.

Reply:
Pushed up this change.